src/component/vcf-breadcrumbs.ts

```diff
@@ -281,55 +286,50 @@ export class VcfBreadcrumbs extends ResizeMixin(ElementMixin(ThemableMixin(Polyl
     ellipsis.style.minWidth = '0';
 
     // Create a popover to show the hidden breadcumbs and add it to the ellipsis element
-    let popover = document.createElement("vaadin-popover");
+    const popover = document.createElement("vaadin-popover");
     popover.setAttribute("for", id);
     popover.setAttribute("overlay-role", "menu");
-    popover.setAttribute('accessible-name-ref', "hidden breadcrumbs"); 
+    popover.setAttribute('accessible-name-ref', "hidden breadcrumbs");
```

Comment:
Consider using `aria-labelledby` instead, and also use `role` instead of `overlay-role`.